BlueprintToCpp/Program.cs

```diff
@@ -87,16 +87,25 @@ public static async Task Main(string[] args)
                 return;
             }
 
+            string zlibPath = config.ZlibPath;
+            if (string.IsNullOrEmpty(zlibPath) || zlibPath.Length < 1)
+            {
+                Console.WriteLine("Missing \"ZlibPath\" in the config.json file.");
+            }
+
             EGame version = config.Version;
             if (string.IsNullOrEmpty(version.ToString()) || version.ToString().Length < 1)
             {
                 Console.WriteLine("Please provide a UE version in the config.json file.");
                 return;
             }
 
+            var provider = InitializeProvider(pakFolderPath, usmapPath, oodlePath, zlibPath, version);
+
             string exeDirectory = AppDomain.CurrentDomain.BaseDirectory;
 
             var provider = InitializeProvider(pakFolderPath, usmapPath, oodlePath, version);
+
```

Comment:
Fixup for merge conflict
```suggestion
            string exeDirectory = AppDomain.CurrentDomain.BaseDirectory;

            var provider = InitializeProvider(pakFolderPath, usmapPath, oodlePath, zlibPath, version);
```